pkg/virt-api/rest/subresource_test.go

```diff
@@ -953,14 +981,17 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 				vm.Status.VolumeRequests = existingVolumeRequests
 			}
 
-			patch, err := generateVMVolumeRequestPatch(vm, volumeRequest)
+			generatedPatch, err := generateVMVolumeRequestPatch(vm, volumeRequest)
 			if expectError {
 				Expect(err).To(HaveOccurred())
-			} else {
-				Expect(err).ToNot(HaveOccurred())
+				Expect(generatedPatch).To(Equal([]byte{}))
```

Comment:
```suggestion
				Expect(generatedPatch).To(BeEmpty())
```

I have a feeling like checking for zero length is cleaner and less error prone?